src/custom/CortiAuth.ts

```diff
@@ -228,6 +310,62 @@ export class Auth extends FernAuth {
         }
     }
 
+    private buildTokenRequestBody(
```

Comment:
I would prefer to have it as utility as well -- I don't see it's being dependable on any internal fields or context. Which also makes it testable (if at some point we would like to test code 😅 )